com.plexamp.Plexamp.yaml

```diff
@@ -40,6 +40,10 @@ modules:
       - type: git
         url: https://github.com/refi64/unappimage
         commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+    cleanup:
```

Comment:
I just pushed an updated build process that drops the requirement for unappimage completely. Could you please rebase and drop this cleanup step?